notebooks/en/agent_change_llm.ipynb

```diff
@@ -0,0 +1,317 @@
+{
```

Comment:
<p>Change "Anything else" to "Next steps" or something like that to encourage the reader to try something else</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/140/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>